specification/cognitiveservices/data-plane/SpeechServices/stable/v2.0/Speech.json

```diff
@@ -0,0 +1,13520 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v2.0",
+    "title": "Speech Services API v2.0",
+    "description": "Speech Services API v2.0.",
+    "contact": {
+      "name": "Cognitive Services - Speech Services",
+      "email": "crservice@microsoft.com"
+    }
+  },
+  "host": "westus.cris.ai",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/api/speechtotext/v2.0/accuracytests": {
+      "get": {
+        "tags": [
+          "Custom Speech accuracy tests:"
+        ],
+        "summary": "Gets the list of accuracy tests for the authenticated subscription.",
+        "operationId": "GetAccuracyTests",
```

Comment:
Operation ID should be of the form NOUN_VERB, the NOUN will become the "operation group" when generating SDK code, basically the name of the class in which all operations which shared it can be under. Ex. AccuracyTest_List